internal/test/e2e/otelglobal/main.go

```diff
@@ -19,6 +20,12 @@ var tracer = otel.Tracer(
 	trace.WithSchemaURL("https://some_schema"),
 )
 
+func setUnUsedTracers() {
```

Comment:
```suggestion
func setUnusedTracers() {
```

Unused is a single word.

Reply:
Fixed